<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orbital viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="openglwidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>700</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reload_pushbutton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>300</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>n</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>l</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Probability</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="M_spinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="L_spinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="N_spinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Orbital properties</string>
     </property>
    </widget>
    <widget class="QSlider" name="probSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>95</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="probLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>95%</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>300</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid divisions</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="xdiv_spinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="ydiv_spinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="zdiv_spinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>openGLWidget</zorder>
   <zorder>reload_pushbutton</zorder>
   <zorder>frame_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>openglwidget</class>
   <extends>QOpenGLWidget</extends>
   <header>hdr/openglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
